website/docs/r/project.html.markdown

```diff
@@ -34,6 +52,12 @@ The following arguments are supported:
     *  TFE versions v202405-1 and later support between 3-40 characters
 * `organization` - (Optional) Name of the organization. If omitted, organization must be defined in the provider config.
 * `description` - (Optional) A description for the project.
+* `tags` - (Optional) A map of key-value tags to add to the project.
+* `ignore_additional_tags` - (Optional) Explicitly ignores `tags`
```

Comment:
If we don't recommend it, should we even include it?

Is there enough precedent elsewhere for this kind of configurability to back up including it in the initial k/v tags release?

Reply:
Yes `ignore_additional_tags` prevents accidentally overriding tags that may have been added outside of configuration. This is helpful in scenarios where you want to ensure certain tags are present on the target, but you do not wish to manage all tags on the target.